setup.py

```diff
@@ -0,0 +1,40 @@
+import pathlib
```

Comment:
Is it possible to put these packaging files in a subdirectory to better organize?

Reply:
it is recommended to keep them at the root of the project, with all of the contained scripts within the same directory

Reply:
Ok, let's expand the README section on the package to describe these files and what they are used for then.